### PR TITLE
S29 al008

### PR DESCRIPTION
**Pull request by FamNuven**
test pull request

### CODE REVIEW

**Comment by zoroMdeD** [comment]
Test miss

File: F407ZGT6_S29AL008/Core/Src/main.c [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+  uint32_t data = 0;
+  HAL_GPIO_WritePin(GPIOB, RESER_Pin|RY_BY_Pin|BYTE_Pin, GPIO_PIN_SET);
+  /* USER CODE END 2 */
+
+  /* Infinite loop */
+  /* USER CODE BEGIN WHILE */
+  while (1)
+  {
+	// write
+	DATA_OUT();
```
Comment (zoroMdeD) [inline F407ZGT6_S29AL008/Core/Src/main.c:97]: That doing the func!?

**Comment by zoroMdeD** [comment]
Test miss

**Review by zoroMdeD — COMMENTED** [review]
Review is ok) test

**Comment by FamNuven** [comment]
?

**Pull request merged**